libpod/runtime_img.go

```diff
@@ -1080,3 +1121,32 @@ func findImageInSlice(images []storage.Image, ref string) (storage.Image, error)
 	}
 	return storage.Image{}, errors.New("could not find image")
 }
+
+// getImageDigest creates an image object and uses the hex value of the digest as the image ID
+// for parsing the store reference
+func getImageDigest(src types.ImageReference, ctx *types.SystemContext) (string, error) {
+	newImg, err := src.NewImage(ctx)
+	if err != nil {
+		return "", err
+	}
+	defer newImg.Close()
+
+	digest := newImg.ConfigInfo().Digest
+	if err = digest.Validate(); err != nil {
+		return "", errors.Wrapf(err, "error getting config info")
+	}
+	return "@" + digest.Hex(), nil
+}
+
+func getPolicyContext(ctx *types.SystemContext) (*signature.PolicyContext, error) {
```

Comment:
Can we get a brief comment on this function?

Reply:
fixed